lib/new.js

```diff
@@ -0,0 +1,687 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
```

Comment:
format: style mismatch:
```javascript
16|
17|'use strict';
18|
```
```suggestion
'use strict';const path = require('path');const puppeteer = require('puppeteer-core');const findChrome = require('./find_chrome');const {rpc} = require('../rpc');const debugApp = require('debug')('carlo:app');const debugServer = require('debug')('carlo:server');const {Color} = require('./color');const {HttpRequest} = require('./http_request');const fs = require('fs');const util = require('util');const {URL} = require('url');const EventEmitter = require('events');const fsReadFile = util.promisify(fs.readFile);let testMode = false;class App extends EventEmitter {/**
```

⏎⏎ at column 14 should be removed.
Triggered rule # 276
```
  +1.roles not in {ARGUMENT, CALLEE}
	∧ +1.reserved not in {=, {}
	∧ +2.length ≥ 3
	∧ +2.reserved not in {(}
	∧ +2.roles not in {RIGHT}
	∧ +3.roles not in {IDENTIFIER}
	∧ +5.internal_type = Identifier
	∧ ^1.roles in {BLOCK} and not in {IDENTIFIER, OPERATOR, DECLARATION}
	∧ ^2.roles not in {CALL}
	∧ -1.roles not in {STRING, EXPRESSION}
	∧ -1.reserved = ; and not in {,, :, ), .}
	∧ -1.diff_offset ≤ 1
	∧ -2.diff_line = 0
	∧ -2.label not in {<-space>}
	∧ -2.diff_col ≤ 3
	∧ -3.label not in {<newline>}
	∧ -4.roles not in {CALL}
	∧ -4.reserved not in {;}
	⇒ y = ⏎
	Confidence: 0.750. Support: 18.
```
⏎ at column 30 should be removed.
Triggered rule # <N/A>
```
  +2.roles in {RIGHT}
	⇒ y = ␣
	Confidence: 0.997. Support: 195.
```